test/Window/Keyboard.test.cpp

```diff
@@ -0,0 +1,66 @@
+#include <SFML/Window/Keyboard.hpp>
+
+// Other 1st party headers
+#include <SFML/System/String.hpp>
+
+#include <catch2/catch_test_macros.hpp>
+
+#include <WindowUtil.hpp>
+
+#include <cctype>
+
+TEST_CASE("[Window] sf::Keyboard", runDisplayTests())
+{
+    SECTION("isKeyPressed(Key)")
+    {
+        CHECK(!sf::Keyboard::isKeyPressed(sf::Keyboard::W));
+        CHECK(!sf::Keyboard::isKeyPressed(sf::Keyboard::A));
+        CHECK(!sf::Keyboard::isKeyPressed(sf::Keyboard::S));
+        CHECK(!sf::Keyboard::isKeyPressed(sf::Keyboard::D));
+    }
+
+    SECTION("isKeyPressed(Scancode)")
+    {
+        CHECK(!sf::Keyboard::isKeyPressed(sf::Keyboard::Scan::W));
+        CHECK(!sf::Keyboard::isKeyPressed(sf::Keyboard::Scan::A));
+        CHECK(!sf::Keyboard::isKeyPressed(sf::Keyboard::Scan::S));
+        CHECK(!sf::Keyboard::isKeyPressed(sf::Keyboard::Scan::D));
+    }
+
+    SECTION("localize(Scancode)")
+    {
+        CHECK(sf::Keyboard::localize(sf::Keyboard::Scan::W) == sf::Keyboard::W);
```

Comment:
This kind of check will fail with some keyboard layouts that are not QWERTY.

Reply:
Good point. I suppose we can't reliably test this then.

Reply:
Are there any keys and scans that always map to each other on all keyboard layouts?

Reply:
Keys like Esc, Space, Enter should match in most cases, but in theory layouts can map to anything else 😄

Reply:
I'm going to use the spacebar as a very conservative choice so that we can continue to have non-zero coverage of these functions. If that's a problem and we're forced to remove these tests then so be it.